Dockerfile

```
@@ -1,11 +1,13 @@
-FROM python:3.9-alpine
+FROM python:3.9-slim
 ARG SHIELDHIT_PATH
 COPY $SHIELDHIT_PATH /usr/local/bin/
 WORKDIR /usr/local/app
-COPY requirements.txt ./
+
+# install dependencies
+COPY ./requirements.txt ./
 RUN apk add gcc musl-dev build-base && \
     pip install -r requirements.txt && \
     apk del gcc musl-dev build-base
```

```suggestion
RUN pip install -r requirements.txt
```

`apk` is not available in `slim` docker images. Moreover its not needed as we will use wheel packages